app/static/playwright.config.ts

```diff
@@ -54,16 +55,4 @@ export default defineConfig({
 
   /* Folder for test artifacts such as screenshots, videos, traces, etc. */
   // outputDir: 'test-results/',
-
-  /* Run your local dev server before starting the tests */
-  webServer: {
-    /**
-     * Use the dev server by default for faster feedback loop.
-     * Use the preview server on CI for more realistic testing.
-     * Playwright will re-use the local server if there is already a dev-server running.
-     */
-    command: process.env.CI ? 'npm run build' : 'npm run dev',
-    port: 5173,
-    reuseExistingServer: !process.env.CI
-  }
```

Comment:
Why isn't this needed any more? On CI it does build and run server so I guess don't need it there? But it's nice to not have to remember to run the app manually when running locally.

Reply:
this was pretty much never needed, the thing is we have to remember to run the app manually when running browser tests anyway, because doing an npm run dev/serve doesnt actually bring up our app, we have to also start our express server, i can include this in a later pr and only have this config if we are on local but we will have to run our build and run script or something like that